scripts/build/get_source/termux_git_clone_src.sh

```diff
@@ -17,6 +17,13 @@ termux_git_clone_src() {
 		git submodule update --init --recursive --depth=1
 		popd
 
+		ACTUAL_CHECKSUM=$(git -c core.abbrev=no -C "$TMP_CHECKOUT" archive --format tar "$TERMUX_PKG_GIT_BRANCH" | sha256sum 2>&1)
+		if [ "$TERMUX_PKG_SHA256" != "${ACTUAL_CHECKSUM%% *}" ]; then
```

Comment:
I would say please use `[[` since this is being run as a bash script.
But de-POSIXism-ing our Bash scripts is a task for another time.